samples/bluetooth/fast_pair/locator_tag/src/ui/platform_dk.c

```diff
@@ -114,7 +120,9 @@ static void btn_handle(uint32_t button_state, uint32_t has_changed)
 			 */
 			if (prev_uptime != 0) {
 				hold_time = (k_uptime_get() - prev_uptime);
-				if (hold_time > RECOVERY_MODE_BTN_MIN_HOLD_TIME_MS) {
+				if (hold_time > DFU_MODE_BTN_MIN_HOLD_TIME_MS) {
```

Comment:
Consider adding a `BUILD_ASSERT` to ensure `DFU_MODE_BTN_MIN_HOLD_TIME_MS > RECOVERY_MODE_BTN_MIN_HOLD_TIME_MS`